feature/experimental/qos/ate_tests/qos_policy_config_test/qos_policy_config_test.go

```diff
@@ -332,32 +365,75 @@ func TestQoSInputIntfClassifierConfig(t *testing.T) {
 		inputClassifierType oc.E_Input_Classifier_Type
 		classifier          string
+		classType           oc.E_Qos_Classifier_Type
+		termID              string
+		dscpSet             []uint8
+		targetGrpoup        string
+		queueName           string
 	}{{
 		desc:                "Input Classifier Type IPV4",
 		inputClassifierType: oc.Input_Classifier_Type_IPV4,
 		classifier:          "dscp_based_classifier_ipv4",
+		classType:           oc.Qos_Classifier_Type_IPV4,
+		dscpSet:             []uint8{0, 1, 2, 3},
+		termID:              "0",
+		targetGrpoup:        "target-group-BE1",
+		queueName:           "0",
 	}, {
 		desc:                "Input Classifier Type IPV6",
 		inputClassifierType: oc.Input_Classifier_Type_IPV6,
 		classifier:          "dscp_based_classifier_ipv6",
+		classType:           oc.Qos_Classifier_Type_IPV6,
+		termID:              "0",
+		targetGrpoup:        "target-group-BE1",
+		dscpSet:             []uint8{0, 1, 2, 3},
+		queueName:           "0",
 	}}
 
 	d := &oc.Root{}
 	q := d.GetOrCreateQos()
 	i := q.GetOrCreateInterface(dp.Name())
 	i.SetInterfaceId(dp.Name())
+	t.Logf("Interface config required to bind a classifier with an interface")
+	i.GetOrCreateInterfaceRef().Interface = ygot.String(dp.Name())
+	i.GetOrCreateInterfaceRef().Subinterface = ygot.Uint32(0)
+	ip := &oc.Interface{}
 
 	t.Logf("qos input classifier config cases: %v", cases)
 	for _, tc := range cases {
 		t.Run(tc.desc, func(t *testing.T) {
+			t.Logf("Classifier config required to bind with an interface")
+			classifier := q.GetOrCreateClassifier(tc.classifier)
+			classifier.SetName(tc.classifier)
+			classifier.SetType(tc.classType)
+			term, err := classifier.NewTerm(tc.termID)
+			if err != nil {
+				t.Fatalf("Failed to create classifier.NewTerm(): %v", err)
+			}
+			term.SetId(tc.termID)
+			action := term.GetOrCreateActions()
+			action.SetTargetGroup(tc.targetGrpoup)
+			condition := term.GetOrCreateConditions()
+			if tc.classifier == "dscp_based_classifier_ipv4" {
+				s := ip.GetOrCreateSubinterface(0).GetOrCreateIpv4()
+				s.Enabled = ygot.Bool(true)
+				condition.GetOrCreateIpv4().SetDscpSet(tc.dscpSet)
+			} else if tc.classifier == "dscp_based_classifier_ipv6" {
+				s := ip.GetOrCreateSubinterface(0).GetOrCreateIpv6()
+				s.Enabled = ygot.Bool(true)
+				condition.GetOrCreateIpv6().SetDscpSet(tc.dscpSet)
+			}
+			fwdGroup := q.GetOrCreateForwardingGroup(tc.targetGrpoup)
+			fwdGroup.SetName(tc.targetGrpoup)
+			fwdGroup.SetOutputQueue(tc.queueName)
 			c := i.GetOrCreateInput().GetOrCreateClassifier(tc.inputClassifierType)
 			c.SetType(tc.inputClassifierType)
 			c.SetName(tc.classifier)
 			gnmi.Replace(t, dut, gnmi.OC().Qos().Config(), q)
 		})
 
 		// TODO: Remove the following t.Skipf() after the config verification code has been tested.
-		t.Skipf("Skip the QoS config verification until it is tested against a DUT.")
+		//t.Skipf("Skip the QoS config verification until it is tested against a DUT.")
```

Comment:
Why comment this out? If it doesn't need to be skipped, remove the line and the TODO

Reply:
fixed